client/core/core_test.go

```diff
@@ -1992,7 +2047,13 @@ func TestTradeTracking(t *testing.T) {
 	qty := 2*matchSize + cancelledQty
 	rate := tBTC.RateStep * 10
 	lo, dbOrder, preImgL, addr := makeLimitOrder(dc, true, qty, tBTC.RateStep)
+	// fundCoinDcrID := encode.RandomBytes(36)
+	// lo.Coins = []order.CoinID{fundCoinDcrID}
 	loid := lo.ID()
+
+	//fundCoinDcr := &tCoin{id: fundCoinDcrID}
+	//tDcrWallet.fundingCoins = asset.Coins{fundCoinDcr}
+
```

Comment:
While `TestHandleRevokeMatchMsg` is updated, improvements to `TestTradeTracking` for better coverage of the `runMatches>negotiate` path are warranted as well since it is using an empty slice of coins for the funding orders.  This is a TODO though since just adding a >0 length coins slice even before the use of `ReturnCoins` adds a lot more to this test that is out of scope for this PR that is meant to fix other bugs.